docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,4 +33,5 @@ Screenshot of unit tests run passing:
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I have researched WCAG2.1 standards and met them in this PR
```

Comment:
Instead of a separate item, can we add it like this:
`- [ ] I have researched WCAG2.1 standards and met them in this PR (can be navigated using a keyboard)

Reply:
One of the items in our backlog is to setup this: https://github.com/pa11y/pa11y

Which means we'll automate some of this and won't need to be that detailed in a checklist.

Reply:
I agree with @caleybrock, good idea. Instead of adding another item in the checklist, we should just specify that we can navigate using a keyboard, as she mentioned.

Reply:
Agreed. @caleybrock do we just want to add your proposed change "(can be navigated using a keyboard)" to #72 and close this one?

Reply:
Agreed @MichaelCherry. We should do that, instead of having two PRs for the **PR template formatting updates**.